### PR TITLE
[TwigBridge] Use a div instead of p to prevent invalid HTML

### PR DESCRIPTION
**Pull request by tijsverkoyen**
| Q             | A
| ------------- | ---
| Branch?       | 6.1
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| License       | MIT

When HTML is provided as the `help`-text with `help_html=true` the HTML should
not be in a `<p>`-element, as this will generate invalid HTML.

A div is also a block element like a p tag.

The `help` for form elements allows for html when `help_html` is set to `true`. But the provided HTML is wrapped in a `<p>` tag. If you provide other block elements (eg: `div`, `ul`, ...) this will output invalid HTML. 
Instead of wrapping it in a `<p>`-tag this PR changes this to a `<div>`-tag.

Another option could be to only use a `<div>`-tag if `help_html === true`.

### CODE REVIEW

File: src/Symfony/Bridge/Twig/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 CHANGELOG
 =========
 
+* Wrap help messages on form elements in `div` instead of `p`.
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Twig/CHANGELOG.md]: please add a 6.1 section title

Comment (tijsverkoyen) [reply src/Symfony/Bridge/Twig/CHANGELOG.md]: Fixed in 90b59320db2ab5f7ad97c0b45866f2b18c0838df

Comment (nicolas-grekas) [reply src/Symfony/Bridge/Twig/CHANGELOG.md]: thanks now I notice another minor change: please add a space at start of line, and remove the final dot

Comment (tijsverkoyen) [reply src/Symfony/Bridge/Twig/CHANGELOG.md]: Sorry, fixed in cc62c2acffe34b38842a324ad9a504fcd2ef7a7c

**Review by javiereguiluz — APPROVED** [review]
Thanks @tijsverkoyen!

What you say it's true: now that we can render any HTML contents here, we cannot keep using `<p>` as the wrapping element.

In practice, this will "break" the design for those using selectors like `p.help-text` instead of just `.help-text` ... but the fix should be a trivial "find and replace" in their editors, so I'd say it's OK to merge this.

**Comment by fabpot** [comment]
@tijsverkoyen Can you have a look at the broken tests?

**Comment by tijsverkoyen** [comment]
@fabpot I will, but I really am struggling. I can't find any references to the old `p`-tag. 

If I understand the [output](https://github.com/symfony/symfony/runs/5597950142?check_suite_focus=true#step:8:7521) of the test correctly it should be fixed in `/home/runner/work/symfony/symfony/src/Symfony/Bridge/Twig/vendor/symfony/form/Tests/AbstractLayoutTest.php:435`. Which is in the vendor-folder, as it is installed through composer. 
It seems like an older version is installed as the file has been fixed: https://github.com/symfony/symfony/pull/45771/files#diff-4125508802c9ee9145cb3e5f9c6742dad7c084623d03442c38b7cba10b8c8f09L435

```1) Symfony\Bridge\Twig\Tests\Extension\FormExtensionDivLayoutTest::testHelp
Failed asserting that 

/div
    [@id="name_help"]
    [@class="help-text"]
    [.="[trans]Help text test![/trans]"]

matches exactly once. Matches 0 times in 

<p id="name_help" class="help-text">[trans]Help text test![/trans]</p>

/home/runner/work/symfony/symfony/src/Symfony/Bridge/Twig/vendor/symfony/form/Tests/AbstractLayoutTest.php:94
/home/runner/work/symfony/symfony/src/Symfony/Bridge/Twig/vendor/symfony/form/Tests/AbstractLayoutTest.php:435
```

**Review by nicolas-grekas — APPROVED** [review]
(I updated the composer.json to hopefully fix failures, but GHA doesn't run at the moment :'( )

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @tijsverkoyen.

**Pull request merged**